tests/fixtures/write-elm-json/elm.input.json

```diff
@@ -0,0 +1,25 @@
+{
+  "nonStandardFieldStart": 1,
+  "type": "application",
+  "source-directories": [ "src" ],
+  "elm-version": "0.19.1",
+  "dependencies": {
+      "direct": { "elm/core": "1.0.0" },
+      "indirect": {}
+  },
+  "nonStandardFieldMiddle": [1, 2, 3],
+  "test-dependencies": {
+    "direct": {
+      "elm/regex": "1.0.0",
+      "elm-explorations/test": "1.2.0"
+    },
+    "indirect": {
+      "elm/html": "1.0.0",
+      "elm/virtual-dom": "1.0.2"
+    }
+  },
+  "nonStandardFieldEnd": {
+    "a": 1,
+    "b": 2
+  }
+}
```

Comment:
```suggestion
}

```

Reply:
What is the thinking here?

Reply:
I like files to have trailing new lines?

Reply:
Ah, of course! I misinterpreted the suggested change as wanting a blank line at the end of the file, but you’re right the ending newline is missing here. I’ve added one now.

Reply:
yeah the github diff UI leaves room for improvement here.